### ISSUE TITLE
account attributes for solicitation; job seekers, models for jobs / feeds / professional profiles

### ISSUE THREAD

**Issue by paigeadelethompson** [mastodon/mastodon#21655]
### Pitch

There's a lot of potential for companies to adopt Mastadon which would compliment the need for things like LinkedIn. It would be nice if there was an opt-in for Mastadon users to build a professional profile around their account and it would help to establish incentive for people to take establishing a sense of identity and maintaining good conduct seriously. 

### Motivation

To my own detriment I am uninterested in establishing a LinkedIn account. Unfortunately, employers consider LinkedIn a "necessity of life" even though not everybody is entitled to participate on the platform. I would be okay with seeing LinkedIn adopt the Mastadon protocol themselves to participate with the rest of the world if they choose to but I don't think they should hold a monopoly on job markets and the concept of a "professional" community. I would imagine most job markets would not want that either but at least with Mastadon they have more flexibility to choose with regard to who can apply for their jobs. In general it would be nice if most social media platforms that fit into the category of a micro-blog could adopt a federated protocol. Ultimately it's possible that most may have to to maintain any value in their platform whether it's Twitter or LinkedIn. 

I will just merely point out that there are plenty of reasons for not wanting to establish a professional profile with LinkedIn, namely with regards to the information they collect about people and their track record for ensuring it's stored in a secure manner. So I guess that kind of raises the question of what is the burden of the rest of the Mastadon community if a company like LinkedIn wants to adopt the Mastadon protocol and participate with the federated community in some capacity? Is it even possible? 

I'm only pointing this out though because there really is no point at least to me in building this idea if there's no possibility of making ends meet. I would just like to know that if LinkedIn did not choose to participate, that it would be to their own detriment and not because "nobody in the Mastadon community anticipated their needs." But they're not the only ones to think of.

**Comment by paigeadelethompson** [comment]
something I saw in the issue backlog that is kind of relevant: https://github.com/mastodon/mastodon/issues/20804

**Comment by eishiya** [comment]
I don't think this is a good fit for a microblogging platform like Mastodon, I feel it would be better suited to a dedicated LinkedIn-like federated service that can support the various other features useful for job seeking and CV management. Mastodon shouldn't try to be an alternative to every platform out there.

**Comment by paigeadelethompson** [comment]
LinkedIn at its core is a microblogging platform. Connections are essentially following/follower subscriptions so I'm not sure that it's distinct as far as microblogging is concerned. The various other features useful for job seeking and CV management are essentially the only ones that are exclusive but Mastadon may yet gains some features for other purposes for which there is overlap. Moreover, I think Mastadon should try to be whatever it needs to be to get people to use it. The fact that it's federated should mean that community operators have the choice to opt-in or out of some features.

**Comment by rbairwell** [comment]
I would suggest this is actually a "solved" issue already to a degree - there are services such as https://fediverse.info/explore/people which allow you to opt in and let people search for your profile by hashtag - so you could add "#lookingforwork" your profile and people searching there for "lookingforwork" would find you.

**Comment by eishiya** [comment]
> I think Mastadon should try to be whatever it needs to be to get people to use it.

People use Mastodon for what it is, and that's a part of what makes Mastodon enjoyable to use. There's a degree of informality that's woven into it that makes is a pleasant way to talk to people. Adding dedicated job features would undermine that, and it would send the wrong message about what Mastodon is and who its target audience is. It would probably drive away as many users as it would attract, because a lot of people come to Mastodon to take a break from work-related things and do not want to see that stuff as a baked-in part of the platform.

The fact that software on the Fediverse is interoperable is what makes it so cool - each service can focus on its niche and be good at that, instead of being a bloated, miserable mess like Facebook. Mastodon does not need to be everything, it just needs to be able to talk to many kinds of things, and it already can via ActivityPub. It sucks that there isn't a good Fedi LinkedIn alternative yet, but it's not Mastodon's job to fix that. Follow connections are not unique to microblogging, either, they're a pretty common part of all kinds of social media, and they're part of the ActivityPub protocol for that reason.

**Comment by nemobis** [comment]
The proposal in the title ("account attributes for solicitation") doesn't seem to have any connection to the pitch/motivation text. I'll comment on the proposal, trying to guess what was meant: this is already possible with badges. Anyone can start a Mastodon instance focused on any of the scopes you mentioned, and hand out badges for the various groups of users.
